static/src/js/zustand/slices/__tests__/createSavedProjectSlice.test.ts

```diff
@@ -236,10 +233,11 @@ describe('createSavedProjectSlice', () => {
           }
         })
 
-        expect(actions.handleError).toHaveBeenCalledTimes(1)
-        expect(actions.handleError).toHaveBeenCalledWith({
+        const { errors } = useEdscStore.getState()
+        expect(errors.handleError).toHaveBeenCalledTimes(1)
+        expect(errors.handleError).toHaveBeenCalledWith({
           action: 'setProjectName',
-          error: 'Mock mutation error',
+          error: new Error('Mock mutation error'),
```

Comment:
why do these have to be instantiated as Errors here?

Reply:
Now that we moved the `HandleErrorParams` to `types.ts`, we set the error type as `Error` (previously it would've been type `unknown`). But this means we needed to standardize the `error` parameter we pass to `handleError`